apps/devhub/views.py

```diff
@@ -1777,8 +1777,23 @@ def api_key(request):
         new_credentials = APIKey.new_jwt_credentials(request.user)
         log.info('new JWT key created: {}'.format(new_credentials))
 
+        send_key_change_email(request.user.email, new_credentials.key)
```

Comment:
`async=True` is a keyword arg that this function accepts. Yes, it will queue a background task to send the email rather than blocking the response until the SMTP connection is made to deliver the email: https://github.com/mozilla/olympia/blob/master/apps/amo/utils.py#L224 I don't actually see other emails being sent like this though.